openBCISimulator.js

```diff
@@ -40,9 +41,23 @@ function OpenBCISimulatorFactory() {
         this.buffer = new Buffer(500);
         // Numbers
         this.sampleNumber = -1; // So the first sample is 0
+        // Objects
+        this.time = {
+            current: 0,
+            start: now(),
+            loop: null,
+            ntp0: 0,
+            ntp1: 0,
+            ntp2: 0,
+            ntp3: 0
+        };
+        console.log('Simulator started at time: ' + this.time.start);
+        console.log('Time board has been running: ' + (now() - this.time.start));
         // Strings
         this.portName = portName || k.OBCISimulatorPortName;
 
+        this.time.start = now();
```

Comment:
Good call, it's done just a little bit above. But this is a start to the ntp sync.